### PR TITLE
Enforcing landscape for shares

### PR DESCRIPTION
**Pull request by itsjoekent**
### What does this PR do?
We're currently hitting the 8mb photo limit on Facebook for one of our campaigns, this PR ensures we always send a standard sized landscape image in our open graph tags.

Tested with a normal campaign and a social override

### CODE REVIEW

**Comment by itsjoekent** [comment]
screen shot 2017-10-13 at 5 09 54 pm

File: app/Entities/Campaign.php [hunk 1]
```diff
@@ -176,6 +176,7 @@ public function jsonSerialize()
             'coverImage' => [
                 'description' => $this->coverImage ? $this->coverImage->getDescription() : '',
                 'url' => get_image_url($this->coverImage),
+                'landscape_url' => get_image_url($this->coverImage, 'landscape'),
```
Comment (weerd) [inline app/Entities/Campaign.php]: why do you have to set this here? Doesn't just specifying a `type` in the `get_image_url()` in the social override work?

Comment (itsjoekent) [reply app/Entities/Campaign.php]: That works if it has a social override, which almost nothing does currently

Comment (weerd) [reply app/Entities/Campaign.php]: ah, jes. ok 🤔

Comment (itsjoekent) [reply app/Entities/Campaign.php]: also this should be camel case, whoops

**Pull request merged**